packages/mobile/src/geth/geth.ts

```diff
@@ -274,6 +308,7 @@ function getStaticNodesFile(nodeDir: string) {
 async function writeStaticNodes(nodeDir: string, enodes: string) {
   console.info(`writeStaticNodes enodes are "${enodes}"`)
   const staticNodesFile = getStaticNodesFile(nodeDir)
+  console.info(`writeStaticNodes static nodes file is ${staticNodesFile}`)
```

Comment:
Nit: Logger instead of console, though I realize the old line above sets the wrong example
```suggestion
  Logger.info('Geth@ writeStaticNodes',`static nodes file is ${staticNodesFile}`)
```